pkg/core/server/connect.go

```diff
@@ -118,6 +118,10 @@ func (c *CoreService) ForwardTransaction(ctx context.Context, req *connect.Reque
 		c.core.logger.Debug("received forwarded tx", zap.Any("tx", req.Msg.Transaction))
 	}
 
+	if c.core.rpc == nil {
```

Comment:
this returning an error is okay because it just means the tx doens't get gossiped by _this_ node right?